### ISSUE TITLE
Ingest Pedalion treebanks

### ISSUE THREAD

**Issue by jacobwegner** [scaife-viewer/beyond-translation-site#36]
_refs https://github.com/scaife-viewer/beyond-translation-site/issues/3_

- [ ]  Help to get additional treebanks ingested in parallel with the [arendongithub/arend-scaife-viewer](https://github.com/arendongithub/arend-scaife-viewer) fork (cc: @arendongithub)
- [ ] Determine how to incorporate localization / functional changes from fork (perhaps on a new issue)
